python/fledge/plugins/north/azure_iot/azure_iot.py

```diff
@@ -89,74 +92,62 @@ def plugin_reconfigure():
 
 
 class AzureNorthPlugin(object):
-    """ North Azure Plugin """
+    """North Azure Plugin"""
 
-    def __init__(self):
+    def __init__(self, config):
         self.event_loop = asyncio.get_event_loop()
+        self.primary_connection_string = config["primaryConnectionString"]["value"]
+        self.mqtt_over_websocket = True if config["websockets"]["value"].lower() in ("true", "false") else False
```

Comment:
why True when config["websockets"]["value"].lower() in ... false?

Reply:
Oh crap. It should simply check with `== true`